Libraries/src/Amazon.Lambda.DynamoDBEvents.SDK.Convertor/Amazon.Lambda.DynamoDBEvents.SDK.Convertor.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\..\..\buildtools\common.props" />
+
+  <PropertyGroup>
+    <Description>Amazon Lambda .NET Core support -  DynamoDBEvents SDK Convertor package.</Description>
+    <TargetFrameworks>netcoreapp3.1;net8.0</TargetFrameworks>
```

Comment:
Since this is a new package lets target just `net8.0` instead of including out of support versions of .NET. At some point we will need to this with the other packages.